packages/grafana-runtime/src/utils/DataSourceWithBackend.ts

```diff
@@ -57,34 +57,37 @@ export class DataSourceWithBackend<
    */
   query(request: DataQueryRequest<TQuery>): Observable<DataQueryResponse> {
     const { intervalMs, maxDataPoints, range, requestId } = request;
-    const orgId = config.bootData.user.orgId;
     let targets = request.targets;
+
     if (this.filterQuery) {
       targets = targets.filter(q => this.filterQuery!(q));
     }
+
     const queries = targets.map(q => {
       let datasourceId = this.id;
+
       if (q.datasource === ExpressionDatasourceID) {
         return {
           ...q,
           datasourceId,
-          orgId,
```

Comment:
@ryantxu I did not get this, why is orgId part of each query? it was not used in https://github.com/grafana/grafana/blob/master/pkg/api/metrics.go#L61. And if used it needs to be added in the backend, the frontend value cannot be trusted, or is it just for tracking/logging?

Reply:
Looks like it can be safely removed.